server/src/api/v1/admin/get-stats.ts

```diff
@@ -44,12 +44,12 @@ async function countTotalEntities(model: statEntityTypes, is_active = true): Pro
                     is_active: is_active
                 }
             })
-            .catch(error => {
+            .catch(() => {
                 return 0;
             });
     } else {
         count = await model.count()
-            .catch(error => {
+            .catch(() => {
                 return 0;
             });
```

Comment:
Shorthand davon wäre
```suggestion
            .catch(() => return 0);
```
Was wiederum abgekürzt werden könnte mit

```suggestion
            .catch(() => 0);
```
(Ungetestet)

https://codeburst.io/javascript-understand-arrow-function-syntax-ab4081bba85b

Reply:
Also nur als nice-to-know aber muss nicht geändert werden :)

Reply:
geändert und getestet